src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSecurity.cs

```diff
@@ -17,7 +17,6 @@ public FileSecurity()
         public FileSecurity(string fileName, AccessControlSections includeSections)
             : base(false, fileName, includeSections, false)
         {
-            Path.GetFullPath(fileName);
```

Comment:
This call was being done inside the constructor. I removed it but I wonder if it was added here to throw in case the full path can't be obtained.

Reply:
Yes, do not remove it. Please add a comment that we're depending on GetFullPath validation.